res/values/strings.xml

```diff
@@ -422,9 +422,9 @@
     <string name="pref_language">Language</string>
     <string name="pref_incognito_keyboard">Incognito keyboard</string>
     <!-- Translators: Keep in mind that this is a Request - it must be clear in the wording that this cannot be enforced. -->
-    <string name="pref_incognito_keyboard_explain">Request keyboard to disable personalized learning</string>
+    <string name="pref_incognito_keyboard_explain">Request keyboard to turn off personalized learning</string>
     <string name="pref_read_receipts">Read receipts</string>
-    <string name="pref_read_receipts_explain">If read receipts are disabled, you won\'t be able to see read receipts from others.</string>
+    <string name="pref_read_receipts_explain">If read receipts are turned off, you won\'t know if your messages were displayed.</string>
```

Comment:
Guessing here it also invalidates prior statuses, as opposed to using "are" (only for future messages).